core/application/impl/app_configuration_impl.cpp

```diff
@@ -289,24 +289,24 @@ namespace kagome::application {
 
     po::options_description blockhain_desc("Blockchain options");
     blockhain_desc.add_options()
-        ("genesis,g", po::value<std::string>(), "required, configuration file path")
+        ("chain,g", po::value<std::string>(), "required, configuration file path")
```

Comment:
Is the short name for it "g" in substrate, too?

Reply:
basically they don't have any one letter command for that, so I will remove it here as well